app/views/contact_methods/_webchat.html.erb

```diff
@@ -1,19 +1,17 @@
-<div class="l-contact-panel">
-  <div class="contact-panel contact-panel__chat-open t-chat" id="js-chat">
-    <h2 aria-level="2" class="contact-panel__heading" role="heading"><span class="icon icon--web-chat"></span><%= t('contact_panels.webchat.title') %></h2>
-    <p class="contact-panel__chat-javascript t-chat-javascript">
-      <%= t('contact_panels.webchat.nojs_html') %></a>.
+<div class="contact-panel contact-panel__chat-open t-chat" id="js-chat">
+  <h2 aria-level="2" class="contact-panel__heading" role="heading"><span class="icon icon--web-chat"></span><%= t('contact_panels.webchat.title') %></h2>
+  <p class="contact-panel__chat-javascript t-chat-javascript">
+    <%= t('contact_panels.webchat.nojs_html') %></a>.
```

Comment:
This text appears to be missing: get a `translation_missing` message in the markup.

Reply:
Fixed